.github/workflows/release.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
       # Setup .npmrc file to publish to GitHub Packages
       - uses: actions/setup-node@v4
         with:
-          node-version: '16'
+          node-version: '20'
```

Comment:
can we use node 22 for CI ?
since this is what we plan to have in production...

Reply:
done here : https://github.com/scality/hdclient/pull/833/commits/f3a01c34f30259c70c9cf04efbbd5a9d0d1d424f#diff-95557d53bf91069e59d82b9d8fcfaf52ec84a762858983edd5ef3c9b3e4c8191L18

Reply:
still on 22 it seems...